For POWER and gcc 8+ we need to set -std=c++11 to avoid 'error: identifier “__ieee128” is undefined'

author: Andrew Edmondson (University Birmingham)

--- arch/Linux-POWER.cuda.orig	2020-08-05 18:51:20.000000000 +0100
+++ arch/Linux-POWER.cuda	2020-08-11 11:24:49.471069513 +0100
@@ -13,5 +13,6 @@
 CUDA=$(CUDAFLAGS) -I. $(CUBINCL) $(CUDAINCL) 
 CUDACC=$(CUDADIR)/bin/nvcc -Xcompiler "-m64"
 
-CUDACCOPTS=-O3 --maxrregcount 48 $(CUDAGENCODE) $(CUDA) -use_fast_math
+# For POWER and gcc 8+ we need to set -std=c++11 to avoid 'error: identifier “__ieee128” is undefined'
+CUDACCOPTS=-O3 --maxrregcount 48 $(CUDAGENCODE) $(CUDA) -use_fast_math -std=c++11
 
--- arch/Linux-POWER-g++.arch.orig	2020-08-11 11:34:22.373336634 +0100
+++ arch/Linux-POWER-g++.arch	2020-08-11 11:33:37.085528433 +0100
@@ -2,7 +2,8 @@
 CHARMARCH = multicore-linux-ppc
 
 CXX = g++ -m64 -std=c++11
-CXXOPTS = -O3 -fexpensive-optimizations -ffast-math 
+# For POWER and gcc 8+ we need to set -std=c++11 to avoid 'error: identifier “__ieee128” is undefined'
+CXXOPTS = -O3 -fexpensive-optimizations -ffast-math -std=c++11
 CC = gcc -m64
 COPTS = -O3 -fexpensive-optimizations -ffast-math
 
